flopy/modflow/mflak.py

```diff
@@ -322,7 +322,7 @@ def __init__(self, model, nlakes=1, ipakcb=None, theta=-1.,
         self.sscncr = sscncr
         self.surfdep = surfdep
         if isinstance(stages, float):
-            stages = np.array(self.nlakes, dtype=np.float) * stages
+            stages = np.array([self.nlakes], dtype=np.float) * stages

```

Comment:
This only works for one lake. For multiple lakes with the same stage, use `stages = np.ones(self.nlakes, dtype=float) * stages`